recipes/ignition-cmake/all/conandata.yml

```diff
@@ -2,7 +2,25 @@ sources:
   "2.5.0":
     url: "https://github.com/ignitionrobotics/ign-cmake/archive/ignition-cmake2_2.5.0.zip"
     sha256: "6d35f015e259ec955f3abda64358a705ce5b3376c3df2ab5f0c17bea4b4e2f68"
+  "2.8.0":
```

Comment:
Do you need all these intermediate versions?

https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-is-the-policy-for-adding-older-versions-of-a-package

Typically we recommend just the lastest

Reply:
Let me try to explain:
*  ignition-cmake is used by both [gazebo-classic ](http://gazebosim.org/#collapseVersion11_0) and [ignition-gazebo.](https://ignitionrobotics.org/home) 
* gazebo classic is in maintainance mode till 2025 with no major releases
*  all developments are happening on ignition-gazebo
* gazebo classic is still being used and that's why it's being maintained.
* Both gazebo and ignition-gazebo have quite a  few downstream dependencies as can be seen from the [ignition project github](https://github.com/ignitionrobotics) page. 
* To my best knowledge there are at least 
   - 12 downstream dependencies for gazebo
   -  16 downstream dependencies for ignition
* I'm not really sure which downstream dependency of gazebo will work/fail with which ignition-cmake and same for the dependencies of igntion-gazebo

* IMO gazebo and ignition-gazebo dependencies are a complete mess and  this might be one reason no one has created a conan recipe for both gazebo and ignition-gazebo yet 

* If it is not a problem then I would like to keep at least two versions of ignition-cmake such that the latest version of both gazebo and ignition-gazebo can ultimately be created: 
  - (1) ignition-cmake-2.5 which is needed for ignition-math-6.7.0 which inturn is needed by gazebo or one of its dependency
  - (2) ignition-cmake-2.10.0 which is needed for igniton-math-6.9.0 which inturn is needed by ignition-gazebo or one of its dependency

I already have a PR for ignition-math (#9569) which fails currently due to missing upstream dependency of ignition-camke with versions higher than 2.8.0 on conan-center artifcatory

What do you think ?

Reply:
I removed the versions 2.8 and 2.9 from the recipe.